Description: fix for node-argparse ≥ 2
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-11-01

--- a/bin/js-yaml.js
+++ b/bin/js-yaml.js
@@ -20,13 +20,18 @@
 
 ////////////////////////////////////////////////////////////////////////////////
 
+const { version } = require('../package.json');
 
 var cli = new argparse.ArgumentParser({
   prog:     'js-yaml',
-  version:  require('../package.json').version,
   addHelp:  true
 });
 
+cli.addArgument(['-v', '--version'], {
+  help: 'Display version',
+  action: 'version',
+  version
+});
 
 cli.addArgument([ '-c', '--compact' ], {
   help:   'Display errors in compact mode',
